substrate/frame/broker/src/tick_impls.rs

```diff
@@ -244,25 +220,33 @@ impl<T: Config> Pallet<T> {
 		let sale_start = now.saturating_add(config.interlude_length);
 		let leadin_length = config.leadin_length;
 		let ideal_cores_sold = (config.ideal_bulk_proportion * cores_offered as u32) as u16;
+		let sellout_price = if cores_offered > 0 {
+			// No core sold -> price was too high -> we have to adjust downwards.
+			Some(new_prices.price)
```

Comment:
So considering that the new model has target price 51x higher than base/min price then if no cores are sold we effectively drop the target price to to 1/51th. If for some reason the there is no interest in buying cores for 2 sales the price will drop to 1/(51*51) so 2601 lower then when we started. Not saying this is wrong but we need to be cautious especially if we include price stability mechanisms for renewals.